stable/rabbitmq/templates/statefulset.yaml

```diff
@@ -80,7 +77,7 @@ spec:
           - name: config-volume
             mountPath: /opt/bitnami/rabbitmq/conf
           - name: data
-            mountPath: /bitnami/rabbitmq
+            mountPath: /opt/bitnami/rabbitmq/var/lib/rabbitmq/
```

Comment:
why was this wrong previously? Also, shouldn't be `/bitnami/rabbitmq/var/lib/rabbitmq/` in any case?

Reply:
we are not using /bitnami at all

Reply:
you're right! I forgot about it